gettsim/data/exogene_renten_daten/lade_renten_daten.py

```diff
@@ -32,78 +29,73 @@ def lade_exogene_renten_daten():
-#     detailed explanation see
-#     https://de.wikipedia.org/wiki/Rentenanpassungsformel
-#     """
-#     # Rentenwert: The monetary value of one 'entgeltpunkt'.
-#     # This depends, among others, of past developments.
-#
-#     # First the Lohnkomponente which depands on the wage development of last years.
-#     lohnkomponente = _lohnkomponente(params, year)
-#     # Second riesterfaktor
-#     riesterfaktor = _riesterfactor(params, year)
-#     # Nachhaltigskeitsfaktor
-#     nachhfaktor = _nachhaltigkeitsfaktor(params, year)
-#
-#     # Rentenwert must not be lower than in the previous year.
-#     renten_factor = lohnkomponente * riesterfaktor * nachhfaktor
-#     rentenwert = params[f"rentenwert_ext_{year - 1}"] * min(1, renten_factor)
-#     return rentenwert
-#
-#
-# def _lohnkomponente(params, year):
-#     """Returns the lohnkomponente for each year. It deppends on the average wages of
-#     the previous years. For details see
-#     https://de.wikipedia.org/wiki/Rentenanpassungsformel
-#     """
-#     return params[f"meanwages_{year - 1}"] / (
-#         params[f"meanwages_{year - 2}"]
-#         * (
-#             (params[f"meanwages_{year - 2}"] / params[f"meanwages_{year - 3}"])
-#             / (
-#                 params[f"meanwages_sub_{year - 2}"]
-#                 / params[f"meanwages_sub_{year - 3}"]
-#             )
-#         )
-#     )
-#
-#
-# def _riesterfactor(params, year):
-#     """This factor returns the riesterfactor, depending on the Altersvorsogeanteil
-#     and the contributions to the pension insurance. For details see
-#     https://de.wikipedia.org/wiki/Rentenanpassungsformel
-#     """
-#     return (100 - params[f"ava_{year - 1}"] - params[f"rvbeitrag_{year - 1}"]) / (
-#         100 - params[f"ava_{year - 2}"] - params[f"rvbeitrag_{year - 2}"]
-#     )
-#
-#
-# def _nachhaltigkeitsfaktor(params, year):
-#     """This factor mirrors the effect of the relationship between pension insurance
-#     receivers and contributes on the pensions. It depends on the rentnerquotienten and
-#     some correcting scalar alpha. For details see
-#     https://de.wikipedia.org/wiki/Rentenanpassungsformel
-#     """
-#     rq_last_year = _rentnerquotienten(params, year - 1)
-#     rq_two_years_before = _rentnerquotienten(params, year - 2)
-#     # There is an additional 'Rentenartfaktor', equal to 1 for old-age pensions.
-#     return 1 + ((1 - (rq_last_year / rq_two_years_before)) * params[f"alpha_{year}"])
-#
-#
-# def _rentnerquotienten(params, year):
-#     """The rentnerquotient is the relationship between pension insurance receivers and
-#     contributes. For details see
-#     https://de.wikipedia.org/wiki/Rentenanpassungsformel
-#     """
-#     return (params[f"rentenvol_{year}"] / params[f"eckrente_{year}"]) / (
-#         params[f"beitragsvol_{year}"]
-#         / (params[f"rvbeitrag_{year}"] / 100 * params[f"eckrente_{year}"])
-#     )
+def _rentenwert_from_2018(params, year):
```

Comment:
@Eric-Sommer: The following code can be used to calculate the `rentenwert` since 2018. What is the difference before 2018. Do have less data here? Could we not just calculate it once and then save it for the following years?

Reply:
Please don't expect too much from the current implementation. Basically, the first function (until 2017) takes the observed real value. The other function calculates it for the given year based on the representative (Soep/eu-silc/evs) sample. In a dynamic setting, you would need to re-calculate this value each year.

Reply:
So the values for `rentnerquotienten`, `nachhaltigkeitsfaktor`, `riesterfactor` and `lohnkomponente` are from the SOEP? The values for `rentenwert` before 2018 are official? Could we get official values for 2019 and 2020? In a second step we could provide functions to calculate these things!

Reply:
No they are all from official sources. For the future, you'd need to get aggregate pensions, average earnings etc. from your data. I updated the data to as recent as possible. Doesn't comply with our yaml style guide, though.

Reply:
Thanks!